src/components/Details/VmDetails/tests/__snapshots__/VmDetails.test.js.snap

```diff
@@ -418,9 +418,7 @@ exports[`<VmDetails /> renders IP addresses correctly 1`] = `
             <dt>
               FQDN
             </dt>
-            <dd>
-              ---
-            </dd>
+            <dd />
```

Comment:
For consistency, this should be still `DASHES` if the FQDN is missing. But it's not a big deal, we can add in a follow-up.